packages/victory-core/src/victory-container/victory-container.js

```diff
@@ -160,7 +166,7 @@ export default class VictoryContainer extends React.Component {
           ref={this.saveContainerRef}
           {...this.getOUIAProps(props)}
         >
-          <svg {...svgProps} style={svgStyle}>
+          <svg {...svgProps} {...safeUserProps} style={svgStyle}>
```

Comment:
Not a blocker - would it make more sense just to add these safe user props to the `svgProps` object so we're not passing in more than one props object? Sometimes prop overrides ad defaults can get a bit confusing when we destructure multiple objects here.

Reply:
You got it!